zdm/data/Surveys/parkes_mb_class_II.dat

```diff
@@ -0,0 +1,26 @@
+FBAR  1382 #MHz
```

Comment:
The missing FRB from parkes_mb_class_I was accidentally put in here, needs to be deleted

Reply:
Will be updated in phase III